docs/apv.md

```diff
@@ -277,8 +277,8 @@ De overtreding beschreven in lid 1 staat bekend als “spam”
 1. Het overvallen/oplichten/bestelen van mensen is verboden, dit mag alleen wanneer een persoon bezig is met criminele activiteiten.
 2. Het is in geen enkele situatie toegestaan om geld van mensen hun bank te stelen of om voertuigen te scammen, om spelers te overvallen, te ontvoeren of negatief te bejegenen bij het gemeentehuis of om facturen en/of andere items te stelen van spelers die bezig zijn met een burgerjob.
 3. Wanneer voertuigen gebruikt worden als betaal middel voor illegale dingen, denk aan loodsen, drugs, wapens, etc. vervalt de bescherming zoals beschreven in lid 2.
-4. Met criminele activiteiten zoals vermeld in lid 1 worden dingen met betrekking tot drugs, overvallen, witwassen etc bedoeld, en dus geen overtredingen.
-5. Wanneer iemand een kogelwerend vest, masker, holster o.i.d. draagt zal dit ook gezien mogen worden als criminele activiteit zoals vermeld in lid 1.
+4. Met criminele activiteiten zoals vermeld in lid 1 worden dingen met betrekking tot drugs, overvallen, witwassen, het dragen van een masker, kogelwerend vest, holster etc, bedoeld, en dus geen overtredingen.
+5. Wanneer iemand een kogelwerend vest, masker, holster o.i.d. draagt zal dit ook gezien mogen worden als criminele activiteit zoals vermeld in lid 4.
```

Comment:
dan kan dit dus weg als het al in lid 4 staat anders staat het er dubbel in

Reply:
Helemaal over het hoofd gezien. Deze is nu ook weggehaald.\